docs/reference/cli.rst

```diff
@@ -172,7 +172,7 @@ Optional command flags:
 
     Just point the ``beet import`` command at a directory of files that are
     already catalogged in your library. Beets will automatically detect this
-    situation and avoid duplicating any items. In this situation, the "copy
+    situation and avoid duplicating any items. A UTF-8 encoding will be enforced on your imported file. In this situation, the "copy
```

Comment:
I actually don't think this needs to be in the docs—to be perfectly honest, 99.9% of users will now no longer need to think about the encoding of the file. (Also, this sentence occurs in the middle of a discussion on a different topic.)

Reply:
Understood, I can incorporate that change!